web-console/src/lib/compositions/health/systemErrors.ts

```diff
@@ -183,16 +193,20 @@ export const extractProgramErrors =
       .returnType<SystemError<any, Report>[]>()
       .with({ RustError: P.any }, (e) => {
         const rustCompilerErrorRegex =
-          /((warning:|error:|error\[[\w]+\]:)[\s\S]+?)\n(\n|(?=( +Compiling|warning:|error:|error\[[\w]+\]:)))/g
+          /((warning:|^error:|error\[[\w]+\]:)[\s\S]+?)\n(\n|(?=( +Compiling|warning:|error:|error\[[\w]+\]:)))/gm
```

Comment:
looks scary.. fwiw there is an option that the rustc compiler just gives out JSON formatted error messages. maybe we should send you that @snkas @ryzhyk

Reply:
Had to use regex until this is implemented